kaldb/src/main/proto/manager_api.proto

```diff
@@ -20,6 +20,8 @@ service ManagerApiService {
   //
   // In the future if only a rate is provided with an empty list the allocation will be automatically assigned.
   rpc UpdatePartitionAssignment(UpdatePartitionAssignmentRequest) returns (UpdatePartitionAssignmentResponse) {}
+
+  rpc RestoreSnapshot(RestoreSnapshotRequest) returns (RestoreSnapshotResponse) {}
```

Comment:
These are `Restore`__`Snapshot`__ requests, but are passed to a __`Replica`__`RestoreService`. I would use consistent terminology for both the GRPC api and the service (ie, are we restoring snapshots, or replicas? Probably the latter, since the replicas are what are being re-created).